Makefile

```diff
@@ -590,7 +590,7 @@ awx-kube-dev-build: Dockerfile.kube-dev
 	    -t $(DEV_DOCKER_TAG_BASE)/awx_kube_devel:$(COMPOSE_TAG) .
 
 ## Build awx image for deployment on Kubernetes environment.
-awx-kube-build: Dockerfile
+awx-kube-build: Dockerfile awx/ui_next/src
```

Comment:
note this make target is not what we actually uses for building awx 

need to make sure that the playbook works

Reply:
@TheRealHaoLiu I believe we need to add `awx/ui_next/src` to the `make ui-release` target.